### PR TITLE
Develop mayerhofen

### PR DESCRIPTION
**Pull request by movinalot**
Update deprecated Azure LB attribute
Update FortiGate cfg with more granular FW policies
Update Azure Availability Set Naming to match SUSE style

### CODE REVIEW

File: azure/modules/fortinet/fortiadc/main.tf [hunk 1]
```diff
@@ -725,9 +725,9 @@ resource "azurerm_lb_backend_address_pool" "lb_backend_address_pool" {
 
   for_each = local.lb_backend_address_pools
 
-  name                = each.value.name
-  resource_group_name = each.value.resource_group_name
-  loadbalancer_id     = each.value.loadbalancer_id
+  name = each.value.name
+  #resource_group_name = each.value.resource_group_name
```
Comment (yeoldegrove) [inline azure/modules/fortinet/fortiadc/main.tf]: please remove

File: azure/modules/fortinet/fortigate/main.tf [hunk 2]
```diff
@@ -910,9 +942,9 @@ resource "azurerm_lb_backend_address_pool" "lb_backend_address_pool" {
 
   for_each = local.lb_backend_address_pools
 
-  name                = each.value.name
-  resource_group_name = each.value.resource_group_name
-  loadbalancer_id     = each.value.loadbalancer_id
+  name = each.value.name
+  #resource_group_name = each.value.resource_group_name
```
Comment (yeoldegrove) [inline azure/modules/fortinet/fortigate/main.tf]: please remove

**Review by yeoldegrove — CHANGES_REQUESTED** [review]
Please remove the 2 old comments.
Otherwise :+1:

File: azure/modules/fortinet/fortiadc/main.tf (hunk 1 above)
Comment (movinalot) [reply azure/modules/fortinet/fortiadc/main.tf]: removed

File: azure/modules/fortinet/fortigate/main.tf (hunk 2 above)
Comment (movinalot) [reply azure/modules/fortinet/fortigate/main.tf]: removed

**Review by yeoldegrove — APPROVED** [review]

**Pull request merged**